gurubu-client/src/app/api/og/route.tsx

```diff
@@ -0,0 +1,55 @@
+              fill="black"
+            />
+          </svg>
+          Simple, fast and practical grooming.
+        </div>
+      ),
+      {
+        width: 1200,
+        height: 630,
+      }
+    );
+  } catch (e: any) {
+    console.error(`${e.message}`);
+    return new Response(`Failed to generate the image`, {
+      status: 500,
```

Comment:
let's discuss here

Is it healthy to return the user directly to 500?

Reply:
I agree btw @mertcanaltin , so I changed this a little bit. When something fails, I also return an ImageResponse but without png data, only text.

Normal condition:
<img width="1512" alt="Screenshot 2024-02-18 at 12 07 38" src="https://github.com/Trendyol/gurubu/assets/65610514/37f18c76-5eaa-4178-ad0c-27803c3601a3">

In error condition:
<img width="1512" alt="Screenshot 2024-02-18 at 12 07 21" src="https://github.com/Trendyol/gurubu/assets/65610514/f7762cc0-7fa2-48a3-bd84-59d211ca767a">

Reply:
@armagandalkiran @aahmetcakir @hanifisenturk Should we show the cause of the error here?